### ISSUE TITLE
Specifying which is the paternal/maternal allele

### ISSUE THREAD

**Issue by dylkot** [abyzovlab/vcf2diploid#2]
In a VCF file with a genotype entry 1|0, does vcf2diploid take the left entry as the paternal allele and right value as the maternal allele? I can't seem to find this in the documentation anywhere. Thus in the above example, the paternal allele would be alternate and the maternal allele would be reference.

Thanks!

**Comment by abyzov** [comment]
Yes, this is correct -- "paternal|maternal". This is the convention used by the 1000 Genomes Project.

**Comment by dylkot** [comment]
Thanks! I appreciate the response.
